docs/en/ingest-management/agent-policies.asciidoc

```diff
@@ -38,6 +38,53 @@ Or, organize policies by functional groupings of how the hosts are
 used: IT email servers, Linux servers, user work-stations, etc.
 Or perhaps by user categories: engineering department, marketing department, etc.
 
+[discrete]
+[[agent-policy-types]]
+== Policy types
+
+In most use cases, {fleet} provides complete central management of {agent}s.
+However some use cases, like running in Kubernetes or using our hosted {ess} on {ecloud},
+require {agent} infrastructure management outside of {fleet}.
+With this in mind, there are two types of {agent} policies:
+
+* **regular policy**: The default use case, where {fleet} provides full central
+management for {agent}s. Users can manage {agent} infrastructure by adding,
+removing, or upgrading {agent}s. They can also manage {agent} configuration by updating
```

Comment:
```suggestion
removing, or upgrading {agent}s. Users can also manage {agent} configuration by updating
```

Reply:
I know it seems repetitious to say "users" again, but "they" has an unclear antecedent here. My brain stumbled.